README.md

```diff
@@ -0,0 +1,161 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Leaderboard <a name="about-project"></a>
+
+**Catalog App** is an interactive digital backend app that allows you to show all items and products in a simple, visual and intuitive way.
+
+## 🛠 Built With <a name="built-with"></a>
+
```

Comment:
- [ ] Nice presentation of your project. However, kindly ensure that your video presentation has a maximum length of 5 minutes. Please endeavor to edit or remake the group video presentation.